pkg/manager/constants.go

```diff
@@ -33,17 +33,11 @@ const (
 	// manager option.
 	DefaultMaxConcurrentReconciles = 1
 
-	// DefaultPodNamespace is the default value for the eponymous manager
-	// option.
-	DefaultPodNamespace = defaultPrefix + "system"
-
 	// DefaultPodName is the default value for the eponymous manager option.
 	DefaultPodName = defaultPrefix + "controller-manager"
 
+	DefaultPodNamespace = defaultPrefix + "system"
```

Comment:
is this is still used in the code base ?

Reply:
yes, it goes into the event recorder. i think a follow up issue is to resolve that, and programatically find the running namespace.